From 13c1f55f00c820885bde90c024b6d55346055e41 Mon Sep 17 00:00:00 2001
From: Mark Adler <madler@alumni.caltech.edu>
Date: Thu, 14 Jun 2018 12:04:58 -0700
Subject: [PATCH] Permit bit 11 of the flags in a zip entry header to be set.

pigz would unfairly reject zip files with bit 11 set. That bit
indicates that the filename and comment fields are encoded using
UTF-8.
---
 pigz.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pigz.c b/pigz.c
index 69239b7..3560529 100644
--- a/pigz.c
+++ b/pigz.c
@@ -2750,7 +2750,7 @@ local int get_header(int save) {
             return -4;              // not a local header
         SKIP(2);
         flags = GET2();
-        if (flags & 0xfff0)
+        if (flags & 0xf7f0)
             return -4;
         method = GET();             // return low byte of method or 256
         if (GET() != 0 || flags & 1)
